pod.go

```diff
@@ -162,12 +165,14 @@ type Sockets []Socket
 // Set assigns socket values from string to a Socket.
 func (s *Sockets) Set(sockStr string) error {
 	sockSlice := strings.Split(sockStr, " ")
+	const expectedSockCount = 4
+	const sockDelimiter = ":"
 
 	for _, sock := range sockSlice {
-		sockArgs := strings.Split(sock, ":")
+		sockArgs := strings.Split(sock, sockDelimiter)
```

Comment:
For the record, I think `strings.Split(sock, ":")` is more readable than `strings.Split(sock, sockDelimiter)` and then go look what `sockDelimiter` is.